nipype/pipeline/engine/nodes.py

```diff
@@ -80,7 +89,7 @@ class Node(EngineBase):
     def __init__(self, interface, name, iterables=None, itersource=None,
                  synchronize=False, overwrite=None, needed_outputs=None,
                  run_without_submitting=False, n_procs=None, mem_gb=0.20,
-                 **kwargs):
+                 run_lock=_null_lock, **kwargs):
```

Comment:
Could you add some documentation how to use this parameter? Possibly with examples?